applications/common/backend/src/main/java/pl/ds/kyanite/common/components/models/CodeSnippetComponent.java

```diff
@@ -40,4 +41,12 @@ public class CodeSnippetComponent {
   @ValueMapValue
   @Getter
   private String code;
+
+  @Inject
+  @Getter
```

Comment:
The annotation could be added on class level, since all fields are "gettable". It would make the code shorter (and easier to see overview of all fields)